docs/lokalise2_permission-template.md

```
@@ -0,0 +1,27 @@
+## lokalise2 permission-template
+
+Manage permission templates for a team
+
+### Synopsis
+
+Manage all templates for permissions inside a team.
+
+### Options
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)